packages/stratocacher/src/wrap.js

```diff
@@ -43,7 +43,7 @@ export default function wrap(opts, func){
 		ttr         = ttl * DEFAULT_TTR_RATIO,
 		layers      = [],
 		before      = _ => Q(_),   // Munge arguments.
-		shouldCache = () => true,  // Should we cache the response from func?
+		transform   = _ => _,      // Transform response from func. undefined won't be cached.
```

Comment:
Is there a name that would be more descriptive of where this lives in the lifecycle?

Reply:
👍